script/install-flynn.tmpl

```diff
@@ -127,7 +127,7 @@ main() {
   local packages=(
     "aufs-tools"
     "iptables"
-    "libvirt-bin"
+    "cgroup-lite"
```

Comment:
Were you able to get rid of the cgroup-lite package by setting up the cgroups in flynn-host?